test/Microsoft.AspNet.Mvc.Testing.Test/ControllerTestHelperTest.cs

```diff
@@ -0,0 +1,198 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using Microsoft.AspNet.Http.Core.Collections;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.Testing.Test
+{
+	public class ControllerTestHelperTest
+	{
+		[Fact]
+		public void ControllerTestability_UsingContextRequestAbortedTest()
+		{
+			// Arrange
+			var controller = new TestabilityController();
+
+			var context = new ControllerTestHelperContext()
+			{
+				RequestAborted = true
+			};
+
+			// Act
+			ControllerTestHelper.Initialize(controller, context);
+			var viewResult = controller.UsingContextRequestAborted();
+
+			// Assert
+			Assert.Null(viewResult);
```

Comment:
What is the correct behavior when requests are aborted?? This Context.RequestAborted is used in many places in MusicStore like https://github.com/aspnet/MusicStore/blob/master/src/MusicStore/Controllers/CheckoutController.cs
I can't find any Http status code for it too.